<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageVPN</class>
 <widget class="QDialog" name="ManageVPN">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>278</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frameTitle">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>32</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: DarkBlue;
color: Yellow;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="lblIcon">
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(&quot;:/libicons/img/OpenVPN64.png&quot;)</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblAction">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>DejaVu Sans</family>
          <pointsize>7</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>VPN Certificates</string>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::MinimumExpanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="lblModel">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>DejaVu Sans</family>
          <pointsize>7</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Model</string>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item alignment="Qt::AlignVCenter">
       <widget class="QLabel" name="lblDateTime">
        <property name="font">
         <font>
          <family>DejaVu Sans</family>
          <pointsize>6</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>2021/03/26 - 10:00:00</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lblTPac">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>On TP/TPAC:</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="buddy">
        <cstring>lstTPac</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="lblUsb">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>On USB Key:</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="buddy">
        <cstring>lstUsb</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QListWidget" name="lstTPac">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="showDropIndicator" stdset="0">
        <bool>false</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::MultiSelection</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QListWidget" name="lstUsb">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::MultiSelection</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="cmdRemove">
       <property name="minimumSize">
        <size>
         <width>104</width>
         <height>36</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>104</width>
         <height>36</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string notr="true">Remove</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/libicons/img/Trash_32.png</normaloff>:/libicons/img/Trash_32.png</iconset>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="cmdAdd">
       <property name="minimumSize">
        <size>
         <width>104</width>
         <height>36</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>104</width>
         <height>36</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string notr="true">Add</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/libicons/img/Back.png</normaloff>:/libicons/img/Back.png</iconset>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frameButtons">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>48</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>48</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>68</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="cmdOk">
        <property name="minimumSize">
         <size>
          <width>104</width>
          <height>36</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>104</width>
          <height>36</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>DejaVu Sans</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string notr="true">Ok</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/libicons/img/OK.png</normaloff>:/libicons/img/OK.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cmdCancel">
        <property name="minimumSize">
         <size>
          <width>104</width>
          <height>36</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>104</width>
          <height>36</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>DejaVu Sans</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string notr="true">Cancel</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/libicons/img/Delete.png</normaloff>:/libicons/img/Delete.png</iconset>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
